sympy/ntheory/ecm.py

```diff
@@ -182,36 +192,56 @@ def _ecm_one_factor(n, B1=10000, B2=100000, max_curve=200, seed=None):
     ==========
 
     n : Number to be Factored
-    B1 : Stage 1 Bound
-    B2 : Stage 2 Bound
+    B1 : Stage 1 Bound. Must be an even number.
+    B2 : Stage 2 Bound. Must be an even number.
     max_curve : Maximum number of curves generated
 
+    Returns
+    =======
+
+    integer : A non-trivial divisor of ``n``. If ``n`` is prime, then ``n``.
```

Comment:
Do you mean "a divisor of ``n`` other than 1"?

Reply:
``n``, if ``n`` is prime, else a nontrivial divisor in range (1, n)

Reply:
Yes. Sorry for my poor English.